content/content-strategy/define-user-content-needs/create-actionable-artefacts/top-tasks/index.yml

```diff
@@ -0,0 +1,27 @@
+pagetitle: Top tasks
+description: Abstract
+weight: 10
```

Comment:
Just as a habit to adopt, do a check of all the topic and sub-topic weights for a module. The `weight` value should be unique for all of them. Do this before the next commit for this module.

Reply:
Steve. Done.